D1673/blas_interface.md

```diff
@@ -85,11 +88,16 @@ Our proposal also has the following distinctive characteristics:
 
 ## Interoperable with other linear algebra proposals
 
-We do not intend to compete with [P1385R1](wg21.link/p1385r1), a
+We believe this proposal is complimentary to [P1385R1](wg21.link/p1385r1), a
 proposal for a linear algebra library presented at the 2019 Kona WG21
 meeting that introduces matrix and vector classes and overloaded
 arithmetic operators.  In fact, we think that our proposal would make
 a natural foundation for a library like what P1385 proposes.
+However, a free function interface - which clearly separates algorithms
```

Comment:
"However, a free function interface -- which clearly separates algorithms from data structures -- more naturally allows for a richer set of operations, such as is provided ...."